Azure/sonic-utilities
# Add show and clear commands for headroom pool watermark

Depends on Azure/sonic-swss#1453

**- What I did**
Added support to query and clear headroom pool watermark counters
Added unit test for the headroom pool watermark counters

**- How I did it**
* Modified watermarkstat script to query/clear headroom pool watermark counters
* Added show and clear commands 

**- How to verify it**
* Send traffic such that it treks into the headroom pool and check for the headroom pool usage using the show command below
* Set polling interval to 30s and issue clear commands mentioned below  and verify that counters are cleared

**- New command output (if the output of a command-line utility has changed)**
Show commands
```
admin@sonic:~$ show headroom-pool watermark 
Headroom pool maximum occupancy:
                 Pool    Bytes
---------------------  -------
ingress_lossless_pool    12480

admin@sonic:~$ show headroom-pool persistent-watermark 
Headroom pool maximum occupancy:
                 Pool    Bytes
---------------------  -------
ingress_lossless_pool    12480
```
Clear commands
```
admin@sonic:~$ sudo sonic-clear headroom-pool watermark 
admin@sonic:~$ sudo sonic-clear headroom-pool persistent-watermark 
```
